### DISCUSSION TITLE
Bank tag tab categories/Tag tab groups

### DISCUSSION THREAD

**Discussion by AFKLgnd** [runelite/runelite#16236] [Ideas]
Ability to make groups of tag tabs that expand into the tabs inside the group.

Ex: Make a group for "bossing" that shows up as 1 tag tab in the sidebar of your bank that you could then put individual tag tabs for raids/bosses into.

**Comment by antero111** [comment]
You could make presets with plugin presets to show/hide e.g. bossing and skilling tabs etc. See https://github.com/antero111/plugin-presets/wiki/Bank-tags
